lib/css/gluegun.css

```diff
@@ -311,9 +311,9 @@ html {
   font-size: 18px; }
 
 body {
-  font-family: "Source Sans Pro", sans-serif;
+  font-family: "Roboto", sans-serif;
```

Comment:
I thought we decided on using Source Sans Pro as per Jayshree's feedback and we made the changes on #7 

Also due to this change, as we don't have the font loaded on `index.erb`, we just see the `sans-serif` font on the browser